eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -49,12 +43,9 @@
   <suppress checks="VisibilityModifier" files="BatchOptions.java"/>
   <suppress checks="VisibilityModifier" files="EventHubClientImpl.java"/>
 
-  <!-- Public API already released with inner assignment-->
+  <!-- Unsure if we can modify this existing code without breaking its functionality -->
```

Comment:
This is not what we want to see in the suppressions file. If you are unsure, speak to someone else in the team to try to get clarity and certainty.